File: js/hittingTheJackpot.js
// Problem# 171
// Create a function that takes in an array (slot machine outcome) and returns true if all elements in the array
// are identical, and false otherwise. The array will contain 4 elements
// https://edabit.com/challenge/hxyvTffvdT4E238CY

const testJackpot = (outcome) => [...new Set(outcome)].length === 1;

module.exports = testJackpot;

File: python/hitting_the_jackpot.py
"""
Hitting the Jackpot

Create a function that takes a list (slot machine outcome) and returns True if all elements in the list are identical, and False otherwise. The list will contain 4 elements.

https://edabit.com/challenge/42YDYc3Pfc9QXpk2x
"""
from typing import List


def test_jackpot(result: List[str]) -> bool:
    return len(set(result)) == 1
